packages/ast-tools/src/route-utils.ts

```diff
@@ -79,6 +79,7 @@ export function bootstrapItem(
 
 export function insertImport(fileToEdit: string, symbolName: string,
                                   fileName: string, isDefault = false): Change {
+  fileName = fileName.replace(/\\/, '/'); // correction in windows
```

Comment:
Actually, could you do a ```typescript
if (process.platform.startsWith('win')) {
  fileName = fileName.replace(/\\/g, '/');
}```

This will also fix the bug that it only replaces the first time this character appears.